### PR TITLE
Creating servlet and test to store accessories entered in front end form (#192)

### PR DESCRIPTION
**Pull request by lzoebaker**
In this PR, I include a servlet to load in accessories to the database from the form. I also included a message that pops up if the filepath you entered is not a valid image source. All input validity checks are done in the front end form.

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/StoreAccessory.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  public void init() {
+    this.gson = new Gson();
+    this.datastore = DatastoreServiceFactory.getDatastoreService();
+    this.accessoryDatabase = new AccessoryDatabase(datastore);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // just to ensure that submit was actually clicked
+    if (request.getParameter(SUBMIT_PARAMETER) != null) {
```
Comment (WillHipschman) [inline src/main/java/com/google/sps/servlets/StoreAccessory.java:45]: this method is used a lot. Maybe move to a common place and refactor this to if(submitWasClicked(request)){ ... }

File: src/test/java/com/google/sps/StoreAccessoryTest.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  /** Tests that the doPost method fetches form parameters, creates appropriate Accessory */
+  @Test
+  public void doPost_withValidInput_addsToDatabase() throws IOException, EntityNotFoundException {
```
Comment (WillHipschman) [inline src/test/java/com/google/sps/StoreAccessoryTest.java:75]: Add a test for if submit was not clicked

File: src/main/java/com/google/sps/servlets/StoreAccessory.java (hunk 1 above)
Comment (DirtGrubDylan) [reply src/main/java/com/google/sps/servlets/StoreAccessory.java:45]: +1, can we put a todo and issue for this?

File: src/test/java/com/google/sps/StoreAccessoryTest.java (hunk 2 above)
Comment (DirtGrubDylan) [reply src/test/java/com/google/sps/StoreAccessoryTest.java:75]: +1

File: src/main/java/com/google/sps/servlets/StoreAccessory.java (hunk 1 above)
Comment (lzoebaker) [reply src/main/java/com/google/sps/servlets/StoreAccessory.java:46]: Done, added todo comment with issue link.

File: src/test/java/com/google/sps/StoreAccessoryTest.java (hunk 2 above)
Comment (lzoebaker) [reply src/test/java/com/google/sps/StoreAccessoryTest.java:81]: Added.

**Pull request merged**